docs/content/preview/explore/observability/active-session-history.md

```diff
@@ -91,47 +91,64 @@ List of wait events by the wait event components.
 
 These are the wait events introduced by YugabyteDB, however some of the following [wait events](https://www.postgresql.org/docs/current/monitoring-stats.html) inherited from PostgreSQL might also show up in the [yb_active_session_history](#yb-active-session-history) view.
```

Comment:
> Is that true?

Yes, because we don't use the storage layer of postgres so the storage layer wait events of postgres won't ever come up

> Why list them?

The doc page of postgres lists all of them